pkg/servicelb/controller.go

```diff
@@ -369,8 +426,8 @@ func (h *handler) newDaemonSet(svc *core.Service) (*apps.DaemonSet, error) {
 					Value: strconv.Itoa(int(port.Port)),
 				},
 				{
-					Name:  "DEST_IP",
-					Value: svc.Spec.ClusterIP,
+					Name:  "DEST_IPs",
```

Comment:
Don't mix case in the environment variable name. Is this going to cause a problem with the klipper-lb image?
```suggestion
					Name:  "DEST_IPS",
```

Reply:
Yes, klipper-lb is expecting `DEST_IPs`. Anyway, let's change it because, you are correct, according to best practices env variables should be upper case

Reply:
https://github.com/k3s-io/klipper-lb/pull/26